packages/core/src/shared/constants.ts

```diff
@@ -189,6 +189,14 @@ export const amazonQHelpUrl = 'https://aws.amazon.com/q/'
 export const amazonQVscodeMarketplace =
     'https://marketplace.visualstudio.com/items?itemName=AmazonWebServices.amazon-q-vscode'
 
+// URLs for Amazon EventBridge Scheduler
```

Comment:
Consider grouping related constants together. You could create a separate object or namespace for EventBridge Scheduler URLs to improve organization and readability.
```suggestion
// URLs for Amazon EventBridge Scheduler
export const eventBridgeSchedulerUrls = {
    createSchedule: '',
    apiGateway: '',
    rolePermissions: ''
}
```